src/main/kotlin/net/ccbluex/liquidbounce/utils/block/BlockExtensions.kt

```diff
@@ -278,7 +281,11 @@ private fun handleActionsOnAccept(
     }
 
     if (onPlacementSuccess()) {
-        player.swingHand(hand)
+        if (!silentSwing) {
```

Comment:
`silentSwing` is taken from Scaffold, but this is a util function which is supposed to be used by other modules and scripts as well and would also apply this modification.

Instead pass silentSwing as additional argument of the function.